packages/ui/src/working-groups/modals/ApplicationDetailsModal/ApplicationDetailsModal.tsx

```diff
@@ -28,6 +30,8 @@ export const ApplicationDetailsModal = React.memo(() => {
     modalData: { applicationId },
   } = useModal<ApplicationDetailsModalCall>()
   const { isLoading, application } = useApplication(applicationId)
+  const history = useHistory()
+  const isMyMembership = useIsMyMembership(application?.applicant?.id || '')
```

Comment:
I think that we have a bug here. This looks valid from the TS POV, but it does not render correctly since we do not query for that variable. I'll create a separate issue for that.

Reply:
See #1742.